src/components/App.tsx

```diff
@@ -2180,6 +2182,19 @@ export class App extends React.Component<any, AppState> {
     }));
   };
 
+  private beforeUnload = (event: BeforeUnloadEvent) => {
+    // TODO: don't show the dialog if there's nothing new since the last save/export?
+    const shouldPrompt =
```

Comment:
I'm not really sure that you need a local variable here. It seems pretty obvious why you're doing this condition.